lib/opentok.js

```diff
@@ -327,6 +327,55 @@ OpenTok = function (apiKey, apiSecret, env) {
   */
   this.listArchives = archiving.listArchives.bind(null, apiConfig);
 
+  /**
+  * Updates a {@link Archive} object with either addStream or removeStream
+  *
+  * @param archiveId {String} The archive ID.
+  *
+  * @param options {Object} An options parameter with either of these two properties properties:
+  *
+  * <ul>
+  *
+  *   <li>
+  *     <code>addStream</code> &mdash; The stream to add to archiving
+  *   </li>
+  *
+  *   <li>
+  *     <code>removeStream</code> &mdash; Teh stream to remove from archiving
+  *   </li>
+  *
+  * </ul>
+  *
+  * If you pass the <code>addStream</code> argument, there are two other optional arguments:
+  *
+  * <ul>
+  *
+  *   <li>
+  *     <code>hasAudio</code> &mdash; Archive the audio stream. Default value is true.
+  *   </li>
+  *
+  *   <li>
+  *     <code>hasAudio</code> &mdash; Archive the video stream. Default value is true.
+  *   </li>
+  *
+  * </ul>
+  *
+  * @param callback {Function} The function to call upon completing the operation. One argument is
+  * passed to the function
+  *
+  * <ul>
+  *
+  *   <li>
+  *      <code>error</code> &mdash; An error object (if the call to the method fails).
+  *   </li>
+  *
+  * </ul>
+  *
+  * @method #patchArchive
```

Comment:
This does not match the proposal: https://github.com/opentok/api-review/pull/232/. The proposal has two methods: `OpenTok.addArchiveStream()` and `OpenTok.removeArchiveStream()`.